### PR TITLE
Add bottom sheet after post

### LINKED ISSUE #14704: Add bottom sheet shown after first post was created
In order to show blogging reminders we need to add a bottom sheet that's shown only once after a post is created.

### PR DESCRIPTION
**Pull request by planarvoid**
Fixes #14704 

This PR adds an empty bottom sheet that's shown after creating a new post. There is a structure in place with a recycler view and adapter to add items to the bottom sheet. The whole functionality is behind a local feature flag.

To test:
- Turn the "Blogging reminders" flag on in App settings/Test feature configuration
- Create a new post
- Notice an empty bottom sheet appears
- Notice it doesn't disappear on rotation
- Notice you can close it

To test:
- Turn the "Blogging reminders" flag off in App settings/Test feature configuration
- Create a new post
- Notice the bottom sheet doesn't appear

## Regression Notes
1. Potential unintended areas of impact
- Nothing

PR submission checklist:

- [x] I have completed the Regression Notes.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WordPress/build.gradle [hunk 1]
```diff
@@ -84,6 +84,7 @@ android {
         buildConfigField "String", "PUSH_NOTIFICATIONS_APP_KEY", '"org.wordpress.android"'
         buildConfigField "boolean", "UNIFIED_COMMENTS_LIST", "false"
         buildConfigField "boolean", "MP4_COMPOSER_VIDEO_OPTIMIZATION", "false"
+        buildConfigField "boolean", "BLOGGING_REMIDERS", "false"
```
Comment (renanferrari) [inline WordPress/build.gradle]: There's a typo here: `REMIDERS` should have been `REMINDERS`.

File: WordPress/src/main/java/org/wordpress/android/ui/bloggingreminders/BloggingReminderBottomSheetFragment.kt [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+                val bottomSheet = sheetDialog?.findViewById<View>(
+                    com.google.android.material.R.id.design_bottom_sheet
+                ) as? FrameLayout
+
+                bottomSheet?.let {
+                    val behavior = BottomSheetBehavior.from(it)
+                    behavior.state = BottomSheetBehavior.STATE_EXPANDED
+                }
+            }
```
Comment (renanferrari) [inline WordPress/src/main/java/org/wordpress/android/ui/bloggingreminders/BloggingReminderBottomSheetFragment.kt]: I know you're probably reusing the logic we already use in other `BottomSheetDialogFragment`s, but I believe this whole block can be replaced with something like:

```kotlin
val sheetDialog = dialog as? BottomSheetDialog
sheetDialog?.apply {
    behavior.state = BottomSheetBehavior.STATE_EXPANDED
}
```

What do you think?

File: WordPress/src/main/java/org/wordpress/android/ui/posts/PostListCreateMenuFragment.kt [hunk 3]
```diff
@@ -27,13 +27,13 @@ class PostListCreateMenuFragment : BottomSheetDialogFragment() {
         container: ViewGroup?,
         savedInstanceState: Bundle?
     ): View? {
-        return inflater.inflate(R.layout.add_content_bottom_sheet, container)
+        return inflater.inflate(R.layout.recycler_view_bottom_sheet, container)
```
Comment (renanferrari) [inline WordPress/src/main/java/org/wordpress/android/ui/posts/PostListCreateMenuFragment.kt:30]: Good call renaming this 👍

**Review by renanferrari — COMMENTED** [review]
Thanks for the changes!

App is working as expected when we add a Post from the My Site screen, but if we do it from the Posts screen, I don't think this works. Not sure if you're planning on adding this feature on a separate PR?

Code is looking pretty good in general, I only left a few minor comments below.

File: WordPress/src/main/java/org/wordpress/android/ui/bloggingreminders/BloggingReminderBottomSheetFragment.kt (hunk 2 above)
Comment (planarvoid) [reply WordPress/src/main/java/org/wordpress/android/ui/bloggingreminders/BloggingReminderBottomSheetFragment.kt]: I've tested it and you're right 👍 . Fixed in 405fb1f

**Comment by planarvoid** [comment]
Thanks for the review @renanferrari , good catch! I was actually planning to do this later and forgot to mention it so I've created an issue to tackle this - https://github.com/wordpress-mobile/WordPress-Android/issues/14761

File: WordPress/build.gradle (hunk 1 above)
Comment (planarvoid) [reply WordPress/build.gradle]: Fixed in dc9f2d4

**Review by renanferrari — APPROVED** [review]
Thanks for the changes! 

LGTM 👍

Feel free to merge this once the conflicts are resolved.

**Pull request merged**